lib/connect_proto/src/order.proto

```diff
@@ -112,5 +113,6 @@ message Order {
   Patient patient = 2;
   Visit visit = 3;
   Order order = 4;
-  google.protobuf.Any subject = 5;
+  Substances substances = 5;
+  google.protobuf.Any subject = 6;
```

Comment:
Technically, can't reassign the [field number](https://developers.google.com/protocol-buffers/docs/proto3#assigning_field_numbers) with protobuf. 

> These field numbers are used to identify your fields in the [message binary format](https://developers.google.com/protocol-buffers/docs/encoding), and should not be changed once your message type is in use

It's _possible_ at this point since we haven't released the API for `Subject`, but as a general practice, you don't want to reassign numbers (although you can reorder them in the file itself just for organization purposes, but in)
```suggestion
  Substances substances = 6;
  google.protobuf.Any subject = 5;
```